prodactivity-frontend/managers/RoutineManager.tsx

```diff
@@ -2,11 +2,9 @@ import RoutineViewModel from '../models/RoutineViewModel';
 import { FetchHelper } from '../helpers';
 
 export default class RoutineManager {
-
     baseURL = 'http://localhost:5000/routine';
```

Comment:
If we don't already have a task for it, we should create one to remove these hardcoded base URLs. Maybe it can be an option passed in at build time?

Reply:
Sure, I'll create a task for different environment urls. What I've seen in the past is a config file holding the base URLs, and an option passed in for whether we're running for dev, qa, prod, etc. which will pull the correct URL